src/app/components/Timestamp/README.md

```diff
@@ -0,0 +1,58 @@
+<!-- Please remove all references to Simorgh from this when moving the Timestamp component to Psammead! -->
+# (package name) &middot; (badges)
+
+## Description
+Displays a timestamp, with suitable semantic markup.
+
+## When to use this component
+For displaying an isolated timestamp, separate from the other content in an article or cell.
+
+## When not to use this component
+When an date or time is to be displayed inline inside a paragraph.
+
+## Installation
+`npm install (package-name)`
+
+## Props
+
+| Argument   | Type   | Required | Default | Example                  |
+|------------|--------|----------|---------|--------------------------|
+| `datetime` | string | Yes      | N/A     | `2006-09-24T05:00-07:00` |
```

Comment:
Maybe we could say "string ([supported formats](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/time#Valid_datetime_Values))"

Reply:
```suggestion
| `datetime` | string ([supported formats](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/time#Valid_datetime_Values)) | Yes      | N/A     | `2006-09-24T05:00-07:00` |
```